packages/core/src/components/FileInput/FileInput.stories.mdx

```diff
@@ -7,16 +7,19 @@ import * as stories from './FileInput.stories';
 
 # FileInput
 
-### Basic FileInput component
+The `FileInput` component enables uploading files from the local storage to the server via the File API. As its value is read-only, it is an uncontrolled component.
```

Comment:
We can just keep `The FileInput component enables uploading files from the local storage.`